ballerina/natives.bal

```diff
@@ -148,10 +186,32 @@ isolated function print(string logLevel, string msg, error? err = (), *KeyValues
             logRecord[k] = v;
         }
     }
+    string logOutput = "";
     if format == "json" {
```

Comment:
Shall we use a constant like `FileFormat.JSON` etc ?

Reply:
Updated in 98b4983